### PR TITLE
Add SWC

### PR DESCRIPTION
**Pull request by ysm-dev**
https://swc.rs

SWC cannot be detected on the client side.

### CODE REVIEW

File: src/technologies/d.json [hunk 1]
```diff
@@ -547,6 +547,10 @@
     ],
     "description": "A modern runtime for JavaScript and TypeScript.",
     "icon": "deno.svg",
+    "implies": [
+      "TypeScript",
+      "SWC"
```
Comment (ysm-dev) [inline src/technologies/d.json:552]: ref
https://deno.land/manual@v1.27.0/typescript/overview#how-does-it-work
https://swc.rs/

File: src/technologies/p.json [hunk 2]
```diff
@@ -3033,6 +3033,9 @@
     "js": {
       "parcelRequire": ""
     },
+    "implies": [
+      "SWC"
```
Comment (ysm-dev) [inline src/technologies/p.json:3037]: ref: https://parceljs.org/blog/beta3/#transpilation-performance

**Comment by ysm-dev** [comment]
+ `Next.js` implies `swc` by default from version `13.0.0` (and doesn't implies Webpack) (https://nextjs.org/docs/advanced-features/compiler)
How to write if the tech implying is different depending on the specific version?

**Comment by nurbek91** [comment]
> * `Next.js` implies `swc` by default from version `13.0.0` (and doesn't implies Webpack) (https://nextjs.org/docs/advanced-features/compiler)
>   How to write if the tech implying is different depending on the specific version?

In my opinion, this cannot be done.

File: src/technologies/s.json [hunk 3]
```diff
@@ -5897,6 +5897,16 @@
     "scriptSrc": "api\\.swagifyapp\\.com/",
     "website": "https://swagifyapp.com"
   },
+  "SWC": {
+    "cats": [
+      19
+    ],
+    "description": "SWC is an extensible Rust-based platform for the next generation of fast developer tools.",
+    "icon": "swc.svg",
```
Comment (nurbek91) [inline src/technologies/s.json:5905]: SWC
32x32

**Review by nurbek91 — APPROVED** [review]

File: src/technologies/s.json (hunk 3 above)
Comment (ysm-dev) [reply src/technologies/s.json:5905]: [3f53db3](https://github.com/wappalyzer/wappalyzer/pull/6946/commits/3f53db3cf3a2abbae82b13f2093d91b3f4b28d6a)

**Pull request merged**